package.json

```diff
@@ -66,6 +68,7 @@
   },
   "dependencies": {
     "case-anything": "^2.1.13",
+    "google-protobuf": "^3.21.2",
```

Comment:
Can we make this a devDependency? I don't think we actually require any `google-protobuf` code to actually invoke the code generator? Or am I missing something?